package/kernel/linux/modules/virt.mk

```diff
@@ -115,3 +115,34 @@ define KernelPackage/vfio-pci/description
 endef
 
 $(eval $(call KernelPackage,vfio-pci))
+
+
+define KernelPackage/iommu_v2
+  SUBMENU:=Virtualization
+  TITLE:=IOMMU Version 2 driver
+  KCONFIG:=\
+     CONFIG_UACCE=n \
+     CONFIG_IOMMU_DEBUGFS=n \
+     CONFIG_INTEL_IOMMU=y \
```

Comment:
Isn't this option building it into the kernel?
Maybe we should activate iommu in the x86/64 target.